frontend/packages/console-dynamic-plugin-sdk/src/app/components/status/Status.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from 'react';
+import {
+  BanIcon,
+  ClipboardListIcon,
+  ExclamationTriangleIcon,
+  HourglassHalfIcon,
+  HourglassStartIcon,
+  NotStartedIcon,
+  SyncAltIcon,
+  UnknownIcon,
+} from '@patternfly/react-icons';
```

Comment:
we should add `@patternfly/react-icons` to shared-modules and `shared-modules-init`

https://github.com/openshift/console/blob/master/frontend/packages/console-dynamic-plugin-sdk/src/shared-modules.ts#L4

https://github.com/openshift/console/blob/master/frontend/packages/console-dynamic-plugin-sdk/src/shared-modules-init.ts#L6

Reply:
@invincibleJai trying to find an answer but still not sure, do we need to list it if it's a dependency of `@patternfly/react-core` ?
now, the build fails with `Failed to parse package dependencies: @patternfly/react-icons
`

Reply:
ah it's part of `@patternfly/react-core` then ignore my comment :)